web-client/src/views/Public/TrialSessions/PublicTrialSessionDetails.tsx

```diff
@@ -131,61 +125,65 @@ function NonMobileOpenCases({
 }) {
   return (
     <React.Fragment>
-      <div className="text-semibold push-right margin-bottom-2">
-        Count: {openCases.length}
+      <div className="text-right margin-bottom-2">
+        <span className="text-semibold">Count: </span>
+        {openCases.length}
       </div>
-      <table
```

Comment:
Not much is actually changing here except for wrapping the table in a div so it can be scrolled.